### PR TITLE
docs: update webhook related instructions

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

Update webhook related posts to use new API endpoint

## Issues resolved by this PR

- [x] [DVN-11551]

## More info

[DVN-11551]: https://devopness.atlassian.net/browse/DVN-11551?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #769 (docs: update webhook related instructions)`: Devopness application `thlmenezes-site.devopness.com` deployment **in-progress** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #769 (docs: update webhook related instructions)`: Devopness application `thlmenezes-site.devopness.com` deployment **completed** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #769 (docs: update webhook related instructions)`: Devopness application `thlmenezes-site.devopness.com` deployment **in-progress** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #769 (docs: update webhook related instructions)`: Devopness application `thlmenezes-site.devopness.com` deployment **completed** on action <https://thlmenezes-site.devopness.com>

**Review by jfoliveira — CHANGES_REQUESTED** [review]

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #769 (docs: update webhook related instructions)`: Devopness application `thlmenezes-site.devopness.com` deployment **in-progress** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #769 (docs: update webhook related instructions)`: Devopness application `thlmenezes-site.devopness.com` deployment **completed** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #769 (docs: update webhook related instructions)`: Devopness application `thlmenezes-site.devopness.com` deployment **in-progress** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #769 (docs: update webhook related instructions)`: Devopness application `thlmenezes-site.devopness.com` deployment **completed** on action <https://thlmenezes-site.devopness.com>

File: docs/applications/deploy-application-using-incoming-hook.md [hunk 1]
```diff
@@ -24,14 +24,14 @@ Before creating our application deploy incoming webhook, we need the `Applicatio
...
-1. Copy the `Application ID` and `Deploy Pipeline ID` from the URL, considering the following URL format:
+1. Copy the `Deploy Pipeline ID` from the URL, considering the following URL format:
     ```bash
     https://app.devopness.com/projects/<project_id>/environments/<environment_id>/applications/<application_id>/pipelines/<pipeline_id>
     ```
 
 </details>
 
-Once you have your `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`), please follow the instructions below to add an incoming webhook to your application that triggers the deploy pipeline:
+Once you have your `Deploy Pipeline ID` (`<pipeline_id>`), please follow the instructions below to add an incoming webhook to your application that triggers the deploy pipeline:
```
[the remark this answers is not in the document]
Comment (thlmenezes) [reply docs/applications/deploy-application-using-incoming-hook.md:34]:
```suggestion
Once you have your `<pipeline_id>`, please follow the instructions below to add an incoming webhook to your application that triggers the deploy pipeline:
```

File: docs/applications/integrate-application-deployments-to-ci-workflows.md [hunk 2]
```diff
@@ -27,30 +27,28 @@ links:
 {% endnote %}
 
 
-1. Take note of the `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`) from the application which you want to watch the action statuses
+1. Take note of the `Deploy Pipeline ID` (`<pipeline_id>`) from the application which you want to watch the action statuses
```
Comment (thlmenezes) [inline docs/applications/integrate-application-deployments-to-ci-workflows.md:30]:
```suggestion
1. Take note of the `<pipeline_id>` from the application which you want to watch the action statuses
```

File: docs/webhooks/create-incoming-webhook.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Webhooks, for now, are an API only feature; so this post will guide you through
 
 ## Step 1: Get the data needed for creating a webhook
 
-1. Copy the `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`) from the application which you want to run pipelines programatically
+1. Copy the `Deploy Pipeline ID` (`<pipeline_id>`) from the application which you want to run pipelines programatically
```
Comment (thlmenezes) [inline docs/webhooks/create-incoming-webhook.md:27]:
```suggestion
1. Take note of the ID (`<pipeline_id>`) of a pipeline that runs the `deploy` operation for the application which you want to run pipelines programatically
```

File: docs/webhooks/create-outgoing-webhook.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ links:
 {% endnote %}
 
 
-1. Take note of the `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`) from the application which you want to watch the action statuses
+1. Take note of the `Deploy Pipeline ID` (`<pipeline_id>`) from the application which you want to watch the action statuses
```
Comment (thlmenezes) [inline docs/webhooks/create-outgoing-webhook.md:30]:
```suggestion
1. Take note of the ID (`<pipeline_id>`) of a pipeline that runs the `deploy` operation for the application which you want to watch the action statuses
```

File: docs/applications/integrate-application-deployments-to-ci-workflows.md [hunk 5]
```diff
@@ -27,30 +27,28 @@ links:
...
 
 1. Take note of the `Target URL` (`<target_url>`), `Request Headers` (`<request_headers>`) and `Request Body` (`<request_body>`) fields according to the source provider where the application' source code is hosted, by following the source provider's instructions on the links bellow:
    - [Bitbucket](https://developer.atlassian.com/cloud/bitbucket/rest/api-group-commit-statuses/#api-repositories-workspace-repo-slug-commit-commit-statuses-build-post)
    - [Github](https://docs.github.com/en/rest/commits/statuses#create-a-commit-status)
    - [Gitlab](https://docs.gitlab.com/ee/api/commits.html#set-the-pipeline-status-of-a-commit)
 
-1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing for the `action.started` event, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`
+1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /pipelines/:id/hooks/outgoing` to create a outgoing for the `action.started` event, replacing `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`
```
Comment (jfoliveira) [inline docs/applications/integrate-application-deployments-to-ci-workflows.md:38]:
```suggestion
1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /pipelines/:id/hooks/outgoing` to create an outgoing webhook for the `action.started` event. In the example below, replace `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>` with the actual values of each parameter before submitting the request.
```

File: docs/applications/integrate-application-deployments-to-ci-workflows.md [hunk 6]
```diff
@@ -73,34 +71,24 @@ links:
...
-             ]
-           }
          ]
        }
      }'
    ```
    {% endraw %}
 
-1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing for the `action.failed` event, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`
+1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /pipelines/:id/hooks/outgoing` to create a outgoing for the `action.failed` event, replacing `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`
```
Comment (jfoliveira) [inline docs/applications/integrate-application-deployments-to-ci-workflows.md:80]:
```suggestion
1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /pipelines/:id/hooks/outgoing` to create an outgoing webhook for the `action.failed` event. In the example below, replace `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>` with the actual values of each parameter before submitting the request.
```

File: docs/applications/integrate-application-deployments-to-ci-workflows.md [hunk 7]
```diff
@@ -139,20 +119,18 @@ links:
 
    - NOTE: the field `request_body.context` needs to be the same for all the action status; this way the same commit status will be updated, instead of creating a new entry for every state.
 
-1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing for the `action.completed` event, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`
+1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /pipelines/:id/hooks/outgoing` to create a outgoing for the `action.completed` event, replacing `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`
```
Comment (jfoliveira) [inline docs/applications/integrate-application-deployments-to-ci-workflows.md:122]:
```suggestion
1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /pipelines/:id/hooks/outgoing` to create an outgoing webhook for the `action.completed` event. In the example below, replace `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>` with the actual values of each parameter before submitting the request.
```

File: docs/webhooks/create-outgoing-webhook.md [hunk 8]
```diff
@@ -45,19 +45,17 @@ links:
    - [Bitbucket](https://developer.atlassian.com/cloud/bitbucket/rest/api-group-pullrequests/#api-repositories-workspace-repo-slug-pullrequests-pull-request-id-comments-post)
    - [Github](https://docs.github.com/en/rest/issues/comments#create-an-issue-comment)
    - [Gitlab](https://docs.gitlab.com/ee/api/notes.html#create-new-issue-note)
-1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing webhook to comment on the Pull Request using the source provider where the application' source code is hosted, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`.
+1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /pipelines/:id/hooks/outgoing` to create a outgoing webhook to comment on the Pull Request using the source provider where the application' source code is hosted, replacing `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`.
```
Comment (jfoliveira) [inline docs/webhooks/create-outgoing-webhook.md:48]:
```suggestion
1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /pipelines/:id/hooks/outgoing` to create an outgoing webhook to comment on the Pull Request using the source provider where the application' source code is hosted. In the example below, replace `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>` with the actual values of each parameter before submitting the request.
```

File: docs/applications/integrate-application-deployments-to-ci-workflows.md [hunk 9]
```diff
@@ -175,25 +153,17 @@ links:
...
-             ]
-           }
          ]
        }
      }'
    ```
    {% endraw %}
 
-1. On your local machine, in a terminal window, run command to list all the application hooks, replacing `<application_id>`.
+1. On your local machine, in a terminal window, run command to list all the application hooks, replacing `<pipeline_id>`.
```
Comment (jfoliveira) [inline docs/applications/integrate-application-deployments-to-ci-workflows.md:162]:
```suggestion
1. On your local machine, in a terminal window, run command to list all the pipeline webhooks, replacing `<pipeline_id>`.
```

File: docs/webhooks/create-incoming-webhook.md [hunk 10]
```diff
@@ -98,10 +90,10 @@ Webhooks, for now, are an API only feature; so this post will guide you through
 
 ## Step 3: Ensure the webhook was created correctly
 
-1. On your local machine, in a terminal window, run command to list all the application hooks, replacing `<application_id>`.
+1. On your local machine, in a terminal window, run command to list all the application hooks, replacing `<pipeline_id>`.
```
Comment (jfoliveira) [inline docs/webhooks/create-incoming-webhook.md:93]:
```suggestion
1. On your local machine, in a terminal window, run command to list all the pipeline webhooks, replacing `<pipeline_id>`.
```

File: docs/webhooks/create-outgoing-webhook.md [hunk 11]
```diff
@@ -74,18 +72,16 @@ links:
...
-            "action.completed",
-            "action.failed"
+            "action.status_changed"
           ]
         }
       }'
     ```
     {% endraw %}
-1. On your local machine, in a terminal window, run command to list all the application hooks, replacing `<application_id>`.
+1. On your local machine, in a terminal window, run command to list all the application hooks, replacing `<pipeline_id>`.
```
Comment (jfoliveira) [inline docs/webhooks/create-outgoing-webhook.md:81]:
```suggestion
1. On your local machine, in a terminal window, run command to list all the pipeline webhooks, replacing `<pipeline_id>`.
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

**Review by jfoliveira — APPROVED** [review]

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #769 (docs: update webhook related instructions)`: Devopness application `thlmenezes-site.devopness.com` deployment **in-progress** on action <https://thlmenezes-site.devopness.com>

**Pull request merged**